scripts/test-deploy.sh

```diff
@@ -0,0 +1,57 @@
+#!/usr/bin/env bash
+
+cmd="${1:-build}"; shift
+
+CLEANUP=()
+at_exit() {
+        for cleanup in "${CLEANUP[@]}"
+        do eval ${cleanup}; done
+}
+trap at_exit EXIT
+
+nixpkgs="$(nix-build nix/sources.nix -A nixpkgs \
+                     --no-out-link )"
+test -n "${nixpkgs}" || { echo "ERROR:  couldn't evaluate 'nixpkgs'" >&2; exit 1; }
+
+nixops="$(nix-build  -I nixpkgs="${nixpkgs}"    \
+                     '<nixpkgs>'     -A nixops  \
+                     --no-out-link)"
+test -n "${nixops}" || { echo "ERROR:  couldn't evaluate 'nixops'" >&2; exit 1; }
+
+nixexpr="$(mktemp --tmpdir deploy-XXXXXX.nix)"
+CLEANUP+=("rm -f ${nixexpr}")
+
+cat >"${nixexpr}" <<EOF
+import <nixops/eval-machine-info.nix> {
+        networkExprs = [
+                "$(realpath deployments/cardano-aws.nix)"
+                "$(realpath physical/physical-shelley-dev.nix)"
```

Comment:
you can also use relative paths here, and just leave this as a normal nix file in the repo